### ISSUE TITLE
warning: could not find UI helper 'git-credential-manager-ui' - install packaging issue?

### ISSUE THREAD

**Issue by abbottbd** [git-for-windows/git#4165]
### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit? 
 - git version 2.39.0.windows.1, 64bit

```
$ git --version --build-options

git version 2.39.0.windows.1
cpu: x86_64
built from commit: de9501c66f93785f7eaf10d02ef08d231eb11688
sizeof-long: 4
sizeof-size_t: 8
shell-path: /bin/sh
feature: fsmonitor--daemon
PS C:\Windows\system32>
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 10.0.19045.2006]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
> type "$env:USERPROFILE\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

-Defaults-
Editor Option: VIM
Custom Editor Path: 
Default Branch Option:  
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Git Pull Behavior Option: Merge
Use Credential Manager: Enabled
Performance Tweaks FSCache: Enabled
Enable Symlinks: Disabled
Enable Pseudo Console Support: Disabled
Enable FSMonitor: Disabled

```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

No

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

Powershell and other 3rd party app integrations

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
simple clone or pull to an onprem azure devops repo 
```
 - What did you expect to occur after running these commands?

prompt for credentials and clone the repo

 - What actually happened instead?

error message (or some 3rd party apps crash)
warning: could not find UI helper 'git-credential-manager-ui'

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

** insert URL here **

This appears to be a problem with how 2.39.0 is including GitCredentialManager in the install.   If you manually go and get the GCM install from their github repo and install it over top of 2.39.0 then everything works as it should.

**Comment by abbottbd** [comment]
an existing defect does exist in the git-credential-manager repo, however it isn't clear which team needs to resolve the issue:
https://github.com/GitCredentialManager/git-credential-manager/issues/993

**Comment by dscho** [comment]
@abbottbd please test the latest [snapshot](https://wingit.blob.core.windows.net/files/index.html).

**Comment by dscho** [comment]
> @abbottbd please test the latest [snapshot](https://wingit.blob.core.windows.net/files/index.html).

@abbottbd Ping? I thought you'd find it crucial to ensure that this gets fixed in the (possibly quite imminent) next Git for Windows version?

**Comment by dscho** [comment]
@abbottbd I plan on releasing a new Git for Windows version soon, and it would be a shame if the bug you reported wasn't fixed in it solely due to lack of testing.

**Comment by dscho** [comment]
/add release note The Git Credential Manager version shipped with Git for Windows v2.39.0 [could not always find its UI helper](https://github.com/git-for-windows/git/issues/4165) which was fixed by [upgrading to a fixed version](https://github.com/git-for-windows/git/issues/4166).

**Comment by dscho** [comment]
/add release note bug The Git Credential Manager version shipped with Git for Windows v2.39.0 [could not always find its UI helper](https://github.com/git-for-windows/git/issues/4165) which was fixed by [upgrading to a fixed version](https://github.com/git-for-windows/git/issues/4166).
